apps/site/src/lib/components/forms/shared/ApplyConfirmation/ConfirmationDetails.tsx

```diff
@@ -61,6 +64,22 @@ export default async function ConfirmationDetails({
 				}
 				isLightVersion
 			/>
+
+			<p>
+				Interested in being a{" "}
+				{otherRoles.map((role, index) => (
+					<span key={role}>
+						<a
+							href={role === "Hacker" ? `/apply` : `/${role.toLowerCase()}`}
+							className="text-[#FBA80A] underline"
+						>
```

Comment:
Suggestion: can you use `<Link>` from `next/link` rather than the standard `<a>` tag so the page doesn't reload when you click the links?